JobManagerExample/JobManagerExample.xcodeproj/project.pbxproj

```diff
@@ -330,10 +330,15 @@
 				MARKETING_VERSION = 200.5.0;
 				PRODUCT_BUNDLE_IDENTIFIER = com.esri.JobManagerExample;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SUPPORTED_PLATFORMS = "iphoneos iphonesimulator xros xrsimulator";
+				SUPPORTS_MACCATALYST = NO;
```

Comment:
I guess Mac Catalyst was never added to this app. I can add it in a following PR.